tests/union_intersection/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -118,4 +129,15 @@ type Props = {
   ) => void
 };
 
+declare class FormData {
+  append(
+    options?:
+      | string
+      | {
```

Comment:
Would this look better as `options?: string | {`?

Reply:
If I remember correctly, I only do the printing for undefined and null. We could extend it. I wanted to err on the safe side.